game/world/objects/npc.cpp

```diff
@@ -1988,7 +1995,8 @@ void Npc::tick(uint64_t dt) {
     }
 
   if(!isDown()) {
-    implLookAt(dt);
+      implLookAtNpc(dt);
```

Comment:
Nit: tabs

Reply:
I hope this is now right, too.

To be fair. I just spend about an hour trying to set your code styling in CLion.
It's just impossible. To many edge-cases and inconsistencies.